src/wp-admin/includes/network.php

```diff
@@ -438,35 +441,36 @@ function network_step2( $errors = false ) {
 		?>
 		<h3><?php esc_html_e( 'Enabling the Network' ); ?></h3>
 		<p><?php _e( 'Complete the following steps to enable the features for creating a network of sites.' ); ?></p>
-		<div class="notice notice-warning inline"><p>
 		<?php
+		$notice_message = '<strong>' . __( 'Caution:' ) . '</strong> ';
+		$notice_args    = array(
+			'type'               => 'warning',
+			'additional_classes' => 'inline',
```

Comment:
The function doesn't accept additional_classes as a string. I missed this in the initial commit, but am committing a follow-up.